README.md

```diff
@@ -8,74 +8,74 @@ The package can be installed as:
 
   1. Add `flames` to your list of dependencies in `mix.exs`:
 
-    ```elixir
-    def deps do
-      [{:flames, "~> 0.2.0"}]
-    end
-    ```
+```elixir
+def deps do
+  [{:flames, "~> 0.2.0"}]
+end
+```
 
   2. Ensure `flames` is started before your application:
 
-    ```elixir
-    def application do
-      [applications: [:flames]]
-    end
-    ```
+```elixir
+def application do
+  [applications: [:flames]]
+end
+```
 
   3. Add configuration to tell `flames` what your repository and (optional) Phoenix Endpoint modules are as well as adding it as a Logger backend:
 
-    ```elixir
-    config :flames,
-      repo: MyPhoenixApp.Repo,
-      endpoint: MyPhoenixApp.Endpoint,
-      table: "errors" # Optional, defaults to "errors"
+```elixir
+config :flames,
+  repo: MyPhoenixApp.Repo,
+  endpoint: MyPhoenixApp.Endpoint,
+  table: "errors" # Optional, defaults to "errors"
 
-    config :logger,
-      backends: [:console, Flames.Logger]
-    ```
+config :logger,
+  backends: [:console, Flames.Logger]
+```
 
   4. Add the following migration:
 
-  ```elixir
-  defmodule MyApp.Repo.Migrations.CreateFlamesTable do
-    use Ecto.Migration
-
-    def change do
-      # Make sure this table name matches the above configuration
-      create table(:errors) do
-        add :message, :text
-        add :level, :string
-        add :timestamp, :datetime # or :utc_datetime if you're using the latest ecto
-        add :alive, :boolean
-        add :module, :string
-        add :function, :string
-        add :file, :string
-        add :line, :integer
-        add :count, :integer
-        add :hash, :string
-
-        add :incidents, :json
-
-        timestamps
-      end
-
-      create index(:errors, [:hash])
-      create index(:errors, [:updated_at])
+```elixir
+defmodule MyApp.Repo.Migrations.CreateFlamesTable do
+  use Ecto.Migration
+
+  def change do
+    # Make sure this table name matches the above configuration
+    create table(:errors) do
+      add :message, :text
+      add :level, :string
+      add :timestamp, :datetime # or :utc_datetime if you're using the latest ecto
```

Comment:
maybe we should flip this now since most people will need this to say `:utc_datetime` now...